### DISCUSSION TITLE
3.9.2

### DISCUSSION THREAD

**Discussion by sashirestela** [sashirestela/simple-openai#218] [Announcements]
## What's Changed
* Support for using custom Json ObjectMapper by @sashirestela in https://github.com/sashirestela/simple-openai/pull/216
* Support to pass an existing Json schema for Structured Outputs by @sashirestela in https://github.com/sashirestela/simple-openai/pull/217

**Full Changelog**: https://github.com/sashirestela/simple-openai/compare/v3.9.1...v3.9.2

<hr /><em>This discussion was created from the release <a href='https://github.com/sashirestela/simple-openai/releases/tag/v3.9.2'>3.9.2</a>.</em>

**Comment by sashirestela** [comment]
@tfrancart, @the-gigi FYI

**Reply by the-gigi** [reply]
Thanks @sashirestela ! That was quick.
